Explorer/Assets/DCL/Utilities/DebugUtils.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Diagnostics;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace DCL.Utilities
+{
+    public class DebugUtils
+    {
+        [Conditional("UNITY_EDITOR")]
+        public static void DrawRaycast(float radius, Vector3 rayOrigin, float rayDistance, bool hasHit, RaycastHit hitInfo)
```

Comment:
I think it's a bit weird to have the `radius` as first param 🤔 What do you think about moving it after `rayDstance`?

Reply:
especially having radius for the raycast 😅 could be quite confusing, right